submissions/vakulinina/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;400;600;700&display=swap" rel="stylesheet">
+  <title>Book your seat</title>
+</head>
+
+<body>
+  <main>
+    <header>
+      <h1>Buy tickets</h1>
+    </header>
+    <article class="movie">
+      <div class="movie-poster"></div>
+      <div class="movie-description">
+        <h2>Alita: Battle Angel</h2>
+        <p>2019 &bull; Action | Sci-Fi</p>
+        <p><b>Synopsys</b></p>
+        <p>A deactivated cyborg's revived, but can't remember anything of her past and goes on a quest to find out who
+          she is.</p>
+      </div>
+    </article>
+    <form action="">
+      <fieldset class="date-selector">
+        <legend>Select date</legend>
+        <label class="date-label">
+          <span class="day">Sun</span><span class="date">25</span><span class="month">May</span>
```

Comment:
Use tag `time` this element represents a specific period in time. Also, don't forget about `datetime` attribute.